pkg/operator/etcdmemberlistercontroller/etcdmemberlistercontroller.go

```diff
@@ -56,23 +57,25 @@ type EtcdMemberListerController struct {
 	lastConsecutiveFailures int
 	operatorClient          v1helpers.OperatorClient
 	etcdClient              etcdcli.EtcdClient
+	controllerFactory       *factory.Factory
```

Comment:
@tjungblu  i followed the same pattern from `StatusSyncer` from `library-go` :D

Reply:
that also works, cool workaround

Reply:
thx a lot 🙏🙏

so i need to add logs, events .. anything else ?